crates/iota-cluster-test/src/config.rs

```diff
@@ -17,7 +17,7 @@ pub enum Env {
 }
 
 #[derive(derive_more::Debug, Parser)]
-#[command(name = "", rename_all = "kebab-case")]
+#[command(name = "")]
```

Comment:
🤔

Reply:
There are two other places where this is done, not sure why

Reply:
It's fine to leave it but maybe keep it in mind, I dunno

Reply:
But what is this actually doing?

Reply:
If you are confident this is doing the same, I think I would prefer if we actually use `no_binary_name = true` then, less confusing IMO

Reply:
I tested in the console and there running commands works the same way with both approaches, but I don't know if it can have different behavior for other things